addon/pods/components/frost-tab/component.js

```diff
@@ -1,21 +1,55 @@
 import Ember from 'ember'
 import layout from './template'
-import FrostTabs from 'ember-frost-tabs/pods/components/frost-tabs/component'
+// import FrostTabs from 'ember-frost-tabs/pods/components/frost-tabs/component'
+import { PropTypes } from 'ember-prop-types'
+
+const {
+  Component
+} = Ember
+
+export default Component.extend({
```

Comment:
@sandersky has been suggesting explicit extension using the PropTypesMixin, I'd like to follow that suggestion (safer, I think we may be too aggressive in using a reopen/extension)